library/Zend/InputFilter/InputInterface.php

```diff
@@ -63,6 +70,12 @@ public function setValidatorChain(ValidatorChain $validatorChain);
     public function setValue($value);
 
     /**
+     * @param  mixed $value
+     * @return self
+     */
+    public function setFallbackValue($value);
```

Comment:
We use this method in [Factory](https://github.com/zendframework/zf2/blob/develop/library/Zend/InputFilter/Factory.php#L231), so we should add a method_exists, or how should be the handling for that.